bin/build-plugin-zip.sh

```diff
@@ -99,6 +99,7 @@ done
 
 # Run the build.
 status "Installing dependencies... 📦"
+npm cache verify
```

Comment:
We've been experiencing some issues like the one at https://github.com/WordPress/gutenberg/runs/3654626964?check_suite_focus=true for other `npm install` and `npm ci` in GitHub Actions. Tracked at https://github.com/WordPress/gutenberg/issues/33424 They seem related to some packages that we point to the GitHub SHA instead of the npm package. When there're network issues, things fall apart.

That was just for context. We can still try doing this to see if it improves the situation for the plugin job.

Reply:
We've tried running the action a number of times and it fails https://github.com/WordPress/gutenberg/actions/runs/1254846943 and https://github.com/WordPress/gutenberg/actions/runs/1254532616